templates/vyatta_vyos_show_interfaces.textfsm

```diff
@@ -1,5 +1,5 @@
 Value INTERFACE (\S+)
-Value List IP_ADDRESS ((?:[A-Fa-f0-9:\.]+\/\d+)|-)
+Value List IP_ADDRESS ((?:[A-Fa-f0-9:\.]+)(\/\d+)?|-)
```

Comment:
Is this something we can simplify the regex for and use `\S+` instead?

Also, could you provide a new `.raw` for this use case and the accompanying `.yml`?

Reply:
@FragmentedPacket ok I updated the regex to do `\S+` and also updated the raw and yml files.